core/crop/structure.py

```diff
@@ -78,17 +80,19 @@ class SensorClass(BASE):
     __tablename__ = SENSOR_TABLE_NAME
 
     # columns
-    id = Column(Integer, primary_key=True, autoincrement=True)
+    id = Column(Integer, primary_key=True)
     type_id = Column(
         Integer,
         ForeignKey("{}.{}".format(SENSOR_TYPE_TABLE_NAME, ID_COL_NAME)),
         nullable=False,
     )
     device_id = Column(Unicode(100), nullable=False)
+    installation_date = Column(DateTime, nullable=False)
```

Comment:
installation_date is unnecessary here. installation_date is set in SensorLocationClass.